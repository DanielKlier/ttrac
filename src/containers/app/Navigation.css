.App-logo {
    height: 20px;
}