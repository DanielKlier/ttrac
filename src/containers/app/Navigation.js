import React from 'react';
import {Nav, Navbar, NavItem} from 'react-bootstrap';
import timeManagementLogo from './time-management.svg';
import {Link} from 'react-router-dom';
import './Navigation.css';

export default props => (
    <Navbar inverse collapseOnSelect>
      <Navbar.Header>
        <Navbar.Brand>
          <Link to="/">
            <img src={timeManagementLogo}
                 alt="ttrac logo" className="App-logo"/>
          </Link>
        </Navbar.Brand>
        <Navbar.Toggle/>
      </Navbar.Header>
      <Navbar.Collapse>
        <Nav>
          <NavItem eventKey={1} href="/">Home</NavItem>
          <NavItem eventKey={2} href="/about-us">About</NavItem>
        </Nav>
      </Navbar.Collapse>
    </Navbar>
)