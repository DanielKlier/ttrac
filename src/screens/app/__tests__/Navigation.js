import React from 'react';
import renderer from 'react-test-renderer';
import Navigation from '../Navigation';
import {MemoryRouter} from 'react-router';

test('it renders the Navigation', () => {
    const component = renderer.create(
        <MemoryRouter>
            <Navigation/>
        </MemoryRouter>
    );

    expect(component.toJSON()).toMatchSnapshot();
});
