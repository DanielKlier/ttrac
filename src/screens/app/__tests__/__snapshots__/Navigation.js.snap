// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders the Navigation 1`] = `
<nav
  className="navbar navbar-inverse"
>
  <div
    className="container"
  >
    <div
      className="navbar-header"
    >
      <a
        className="navbar-brand"
        href="/"
        onClick={[Function]}
      >
        <img
          alt="ttrac logo"
          className="App-logo"
          src="time-management.svg"
        />
      </a>
      <button
        className="navbar-toggle collapsed"
        onClick={[Function]}
        type="button"
      >
        <span
          className="sr-only"
        >
          Toggle navigation
        </span>
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
        <span
          className="icon-bar"
        />
      </button>
    </div>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse"
    >
      <ul
        className="nav navbar-nav"
        role={null}
      >
        <li
          className="active"
          role="presentation"
          style={Object {}}
        >
          <a
            href="/"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="/projects"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Projects
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="/about-us"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            About
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}
        >
          <a
            href="/testground"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Testground
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
